public/js/common/export/export.js

```diff
@@ -17,11 +17,16 @@ function createExportModalDiv() {
     var img = (context === 'graph') ? getGraphImage() : getGridImage(session);
     var contentDiv = $('<div></div>');
     var topContentDiv = $('<div></div>');
-    var calendarOption = $('<a href="calendar">Download ICS</a>')
+    var calendarOption = $('<a href="calendar">Download ICS</a>');
+    var gridOption = $('<a href="pdf">Export to PDF</a><br>');
```

Comment:
This URL has no query parameters, but the server expects some. You can generate them yourself and put them in the URL here if you want - just repeat the code in `getGridImage` and call `encodeURIComponent` to encode them to create a URL.